src/docs/components/site-header.svelte

```diff
@@ -40,7 +40,7 @@
 			<a
 				href={siteConfig.links.discord}
 				target="_blank"
-				rel="noreferrer"
+				rel="noopener noreferrer"
```

Comment:
If I am not mistaken, `noreferrer` alone would actually be enough, but `eslint-plugin-svelte` wants `noopener` to be specified too.

Reply:
maybe the plugin wasn't updated after https://github.com/sveltejs/svelte/pull/8230 was merged. might want to file an issue with `eslint-plugin-svelte`

Reply:
I'm fine with it 😄

Reply:
https://github.com/sveltejs/eslint-plugin-svelte/issues/571